### PR TITLE
Update QGIS backend documentation

### PR DESCRIPTION
**Pull request by yjacolin**

### CODE REVIEW

File: doc/integrator/backend_qgis.rst [hunk 1]
```diff
@@ -57,45 +58,64 @@ Remove the mapserver folder:
...
+You should setup your OWS service in the QGIS project properties in the OWS
+tab.
 
-The first solution is to create a tunnel with ssh:
+You should take care to **uncheck** the checkbox *User Layer ids as names*. If
+this checkbox is enabled you will have unreliabel layers name.
 
-.. prompt:: bash
+You should **check** the checkbox *Add geometry to feature response* in order
+to make working the WMS GetFeatureInfo correctly.
```
Comment (ochriste) [inline doc/integrator/backend_qgis.rst]: to make the WMS GetFeatureInfo working correctly.
?

File: doc/integrator/backend_qgis.rst [hunk 2]
```diff
@@ -22,8 +22,8 @@ In the ``cgxp_print`` plugin add the following configuration:
 
 .. code:: javascript
 
-   encodeLayer: { useNativeAngle: true, serverType: 'qgis' }
-   encodeExternalLayer: { useNativeAngle: true, serverType: 'qgis' }
+   encodeLayer: { useNativeAngle: true, serverType: 'QGISSERVER' }
+   encodeExternalLayer: { useNativeAngle: true, serverType: 'QGISSERVER' }
```
Comment (sbrunner) [inline doc/integrator/backend_qgis.rst]: In the documentation it's written in lower case, it's not the case?
http://mapfish.github.io/mapfish-print-doc/layers.html#Wms%20Layer

File: doc/integrator/backend_qgis.rst [hunk 3]
```diff
@@ -46,7 +46,8 @@ In the ``vars_<project>.yaml`` file, define:
 
 .. code:: yaml
 
-    mapserv_url: http://your.arcgis.server/arcgis/services/SOME/FOLDER/MapServer/WMSServer
+    mapserv_url: http://your.qgis.server/ows
+    geoserver: false
```
Comment (sbrunner) [inline doc/integrator/backend_qgis.rst]: It should be false by default => should be fixed in the code ...

File: doc/integrator/backend_qgis.rst [hunk 4]
```diff
@@ -57,45 +58,64 @@ Remove the mapserver folder:
...
+
+Finally, your QGIS project layers CRS should all be in the same CSR. This is subject to
+change.
+
+Connect to Postgres database
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This section is subject to change when the QGIS plugin is available.
+
+* Throw a tunnel
```
Comment (sbrunner) [inline doc/integrator/backend_qgis.rst]: It's not better to use title instance of list?

File: doc/integrator/backend_qgis.rst (hunk 2 above)
Comment (yjacolin) [inline doc/integrator/backend_qgis.rst]: The first attemps with 'qgis' throw an error message giving the value list available: MAPSERVER,GEOSERVER,QGISSERVER

File: doc/integrator/backend_qgis.rst (hunk 3 above)
Comment (yjacolin) [inline doc/integrator/backend_qgis.rst]: strange as it already false. I need to do more test.

File: doc/integrator/backend_qgis.rst (hunk 4 above)
Comment (yjacolin) [inline doc/integrator/backend_qgis.rst]: I agree with you but didn't know with caracters used. In other way, too much title in a page make it not so readable.

File: doc/integrator/backend_qgis.rst (hunk 1 above)
Comment (yjacolin) [inline doc/integrator/backend_qgis.rst]: which one do you prefer? The first proposition was my first attemp but Google Translate prefered mine :D

File: doc/integrator/backend_qgis.rst (hunk 4 above)
Comment (sbrunner) [inline doc/integrator/backend_qgis.rst]: Proposed structure:
```rst
=========================================
Specific configuration for QGIS mapserver
=========================================

Viewer configuration
====================

Apache configuration
====================

Application configuration
=========================

Cleanup
=======

QGIS Desktop configuration
==========================

OWS configuration
-----------------

Connect to Postgres database
----------------------------

Throw a tunnel
~ ~~~~~~~~~~~~~

Directly on the server
~ ~~~~~~~~~~~~~~~~~~~~~

Deploy notes
============
```

I add some spaces to make it working in the comment :-)

File: doc/integrator/backend_qgis.rst (hunk 3 above)
Comment (yjacolin) [inline doc/integrator/backend_qgis.rst]: There is a problem with this https://github.com/camptocamp/c2cgeoportal/blob/master/c2cgeoportal/scaffolds/update/CONST_vars.yaml_tmpl#L209 as it doesn't seem to be used. If "geoserver" is missing in the project vars file, log throw error like this:
```
[Wed Apr 27 15:54:20.006889 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503] Traceback (most recent call last):
[Wed Apr 27 15:54:20.006905 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/fanstatic/publisher.py", line 219, in __call__
[Wed Apr 27 15:54:20.006941 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     return self.app(environ, start_response)
[Wed Apr 27 15:54:20.006960 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/fanstatic/injector.py", line 54, in __call__
[Wed Apr 27 15:54:20.006980 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     response = request.get_response(self.app)
[Wed Apr 27 15:54:20.007000 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/webob/request.py", line 1320, in send
[Wed Apr 27 15:54:20.007024 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     application, catch_exc_info=False)
[Wed Apr 27 15:54:20.007043 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/webob/request.py", line 1284, in call_application
[Wed Apr 27 15:54:20.007063 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     app_iter = application(self.environ, start_response)
[Wed Apr 27 15:54:20.007076 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/pyramid/router.py", line 243, in __call__
[Wed Apr 27 15:54:20.007098 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     response = self.invoke_subrequest(request, use_tweens=True)
[Wed Apr 27 15:54:20.007118 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/pyramid/router.py", line 218, in invoke_subrequest
[Wed Apr 27 15:54:20.007139 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     response = handle_request(request)
[Wed Apr 27 15:54:20.007156 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/pyramid/tweens.py", line 21, in excview_tween
[Wed Apr 27 15:54:20.007200 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     response = handler(request)
[Wed Apr 27 15:54:20.007213 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/pyramid_tm/__init__.py", line 94, in tm_tween
[Wed Apr 27 15:54:20.007234 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     reraise(*exc_info)
[Wed Apr 27 15:54:20.007253 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/pyramid_tm/__init__.py", line 75, in tm_tween
[Wed Apr 27 15:54:20.007273 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     response = handler(request)
[Wed Apr 27 15:54:20.007290 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/pyramid/router.py", line 164, in handle_request
[Wed Apr 27 15:54:20.007310 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     response = view_callable(context, request)
[Wed Apr 27 15:54:20.007329 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/c2cgeoportal/__init__.py", line 136, in new_f
[Wed Apr 27 15:54:20.007351 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     return f(root, request)
[Wed Apr 27 15:54:20.007369 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/pyramid/config/views.py", line 353, in rendered_view
[Wed Apr 27 15:54:20.007395 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     result = view(context, request)
[Wed Apr 27 15:54:20.007418 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/pyramid/config/views.py", line 481, in _class_requestonly_view
[Wed Apr 27 15:54:20.007443 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     response = getattr(inst, attr)()
[Wed Apr 27 15:54:20.007462 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/c2cgeoportal/views/entry.py", line 915, in get_cgxp_viewer_vars
[Wed Apr 27 15:54:20.007490 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     themes, errors = self._themes(role_id, interface)
[Wed Apr 27 15:54:20.007510 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/dogpile/cache/region.py", line 1040, in decorate
[Wed Apr 27 15:54:20.007536 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     should_cache_fn)
[Wed Apr 27 15:54:20.007555 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/dogpile/cache/region.py", line 651, in get_or_create
[Wed Apr 27 15:54:20.007581 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     async_creator) as value:
[Wed Apr 27 15:54:20.007599 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/dogpile/core/dogpile.py", line 158, in __enter__
[Wed Apr 27 15:54:20.007621 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     return self._enter()
[Wed Apr 27 15:54:20.007639 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/dogpile/core/dogpile.py", line 98, in _enter
[Wed Apr 27 15:54:20.007669 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     generated = self._enter_create(createdtime)
[Wed Apr 27 15:54:20.007689 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/dogpile/core/dogpile.py", line 149, in _enter_create
[Wed Apr 27 15:54:20.007713 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     created = self.creator()
[Wed Apr 27 15:54:20.007731 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/dogpile/cache/region.py", line 619, in gen_value
[Wed Apr 27 15:54:20.007757 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     created_value = creator()
[Wed Apr 27 15:54:20.007776 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/dogpile/cache/region.py", line 1036, in creator
[Wed Apr 27 15:54:20.007796 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     return fn(*arg, **kw)
[Wed Apr 27 15:54:20.007815 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/c2cgeoportal/views/entry.py", line 589, in _themes
[Wed Apr 27 15:54:20.007836 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     role_id if self.mapserver_settings["geoserver"] else None
[Wed Apr 27 15:54:20.007865 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503] KeyError: 'geoserver'
```

Comment (sbrunner) [inline doc/integrator/backend_qgis.rst]: should be:
```
  .. code:: yaml
 
     vars:
         ...
         mapserverproxy:
             mapserv_url: http://your.qgis.server/ows
     update_paths:
     ...
     - mapserverproxy
```

File: doc/integrator/backend_qgis.rst [hunk 5]
```diff
@@ -1,10 +1,10 @@
 .. _integrator_backend_qgis:
-
```
Comment (sbrunner) [inline doc/integrator/backend_qgis.rst]: Missing newline?

**Comment by yjacolin** [comment]
\o/ all checks are green :)

**Comment by sbrunner** [comment]
Looks good :-)

**Pull request merged**